### PR TITLE
Pressing enter to submit the Super Admin creation form in the Setup Wizard and show form validation error #11465

### PR DESCRIPTION
**Pull request by a6ar55**
-displays form validation error when enter is clicked.(Password do not match)

https://github.com/learningequality/kolibri/assets/117556787/88eb3a64-627a-4a30-934b-5a31310182e4

## References

This fixes issue #11465 

## Reviewer guidance

----

## Testing checklist

- [x] Contributor has fully tested the PR manually
- [x] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

## PR process

- [ ] PR has the correct target branch and milestone
- [ ] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Comment by MisRob** [comment]
Hi @a6ar55 , thank you for your contribution! We will definitely follow-up, there's been lots of pull requests in our queue.

File: kolibri/core/assets/src/views/userAccounts/PasswordTextbox.vue [hunk 1]
```diff
@@ -122,11 +122,14 @@
         this.$refs.password.focus();
       },
       checkErrorsAndSubmit(e) {
-        if (this.valid) {
-          this.$emit('submitNewPassword');
-        } else {
-          // Blurring will cause validation errors to show if needed
-          e.target.blur();
+        if (e.key === 'Enter') {
```
Comment (jredrejo) [inline kolibri/core/assets/src/views/userAccounts/PasswordTextbox.vue]: I don't quite understand this. If `checkErrorsAndSubmit` is triggered by the the `@keydown.enter="checkErrorsAndSubmit"` event handler, why do you need to check in the event key is 'Enter'?  https://vuejs.org/guide/essentials/event-handling.html#key-modifiers

Comment (a6ar55) [reply kolibri/core/assets/src/views/userAccounts/PasswordTextbox.vue]: Sorry, you're right. It was a mistake. I will fix it.

Comment (a6ar55) [reply kolibri/core/assets/src/views/userAccounts/PasswordTextbox.vue]: @jredrejo I have corrected it.

**Comment by MisRob** [comment]
Hi @a6ar55, thanks for following-up. We will re-review.

**Review by jredrejo — APPROVED** [review]
Code look good to me. 
Tested and working fine.

**Pull request merged**